tests/context.py

```
@@ -380,6 +380,108 @@
+		sys.executable  # Allow the current Python interpreter
+	}
+	command = str(args[0]).lower()
+	# Check for coverage command pattern
+	if any(cmd in command for cmd in ["coverage", sys.executable]):
+		# Special handling for coverage commands
+		return checkCovCommand(*args)
+	if not any(cmd in command for cmd in allowed_commands):
+		raise CommandExecutionError(
+			f"Command '{command}' is not allowed. Allowed commands: {allowed_commands}",
+			exit_code=77
+		)
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)